scss/bitstyles/utilities/font/font.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Utilities/Font" />
+
+# Font
+
+Set font properties of an element.
+
+## Font Weight
+
+<Canvas isColumn>
+  <Story name="font--light">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-font--light">u-font--light. Pastry marzipan gummi bears lollipop pie pie carrot cake dessert.</div>
+      </div>
+    `}
+  </Story>
+  <Story name="font--normal">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-font--normal">u-font--normal. Pastry marzipan gummi bears lollipop pie pie carrot cake dessert.</div>
+      </div>
+    `}
+  </Story>
+  <Story name="font--medium">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-font--medium">u-font--medium. Pastry marzipan gummi bears lollipop pie pie carrot cake dessert.</div>
+      </div>
+    `}
+  </Story>
+  <Story name="font--bold">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-font--bold">u-font--bold. Pastry marzipan gummi bears lollipop pie pie carrot cake dessert.</div>
+      </div>
+    `}
+  </Story>
+</Canvas>
+
+## Font style
+
+<Canvas isColumn>
+  <Story name="font--italic">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-font--italic">u-font--italic. Pastry marzipan gummi bears lollipop pie pie carrot cake dessert. Pie liquorice marshmallow muffin sweet roll gingerbread candy canes. Tiramisu cotton candy bonbon gingerbread brownie pastry. Croissant apple pie croissant halvah lollipop chocolate cake candy canes liquorice marshmallow.</div>
+      </div>
+    `}
+  </Story>
+  <Story name="font--not-italic">
+    {`
+      <div class="u-bg--brand-2 u-padding-m">
+        <div class="u-bg--background u-font--normal">u-font--normal. Pastry marzipan gummi bears lollipop pie pie carrot cake dessert. Pie liquorice marshmallow muffin sweet roll gingerbread candy canes. Tiramisu cotton candy bonbon gingerbread brownie pastry. Croissant apple pie croissant halvah lollipop chocolate cake candy canes liquorice marshmallow.</div>
```

Comment:
Huh actually it appears `font--not-italic` and `u-font--normal` are mixed up over there 🍸

Reply:
Amazing, thanks! This is fixed, it’s now using `u-font--not-italic` 🎉